.github/workflows/update-vulnerabilties.yaml

```diff
@@ -2,6 +2,9 @@ name: Update Vulnerabilities
 on:
   schedule:
     - cron: "0 */3 * * *"
+  push:
+    branches:
+      - yli3/*
```

Comment:
Nit: Maybe we could leave a branch namespace here like `scanner/*`  for testing instead of your own namespace?